build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.props

```diff
@@ -8,4 +8,11 @@
     <WindowsAppSdkFoundation>true</WindowsAppSdkFoundation>
   </PropertyGroup>
 
+  <!-- At least for now, we want MRT Core to not be used in WAP or UWP projects. There, we want the in-box PRI generation tooling to be used. -->
+  <PropertyGroup>
+    <EnableCoreMrtTooling Condition="'$(EnableCoreMrtTooling)'=='' and '$(WindowsAppContainer)'!='true' and '$(MSBuildProjectExtension)'!='.wapproj'">true</EnableCoreMrtTooling>
```

Comment:
It seems like this condition won't entirely work now that it's moved to a .props file.  The `EnableCoreMrtTooling` and `WindowsAppContainer` properties wouldn't be set yet.

Reply:
Excellent find...

When I moved the property definition, that didn't occur to me. The import then worked in my verification, so I didn't question it. I see now that the reason it worked is that my conditions check if the properties are not equal to fixed strings. Empty strings will of course satisfy that.

Will fix!

I wish MSBuild logs captured where a property was assigned its value, not just what its value is. That would have helped in figuring out why this import was working.

Reply:
[MSBuild binary logs](https://msbuildlog.com) actually do capture the locations where properties are reassigned.  I'm not sure if that tells you where it was first set, but if it is changed then it will show you where.

![image](https://user-images.githubusercontent.com/145043/132929974-31b43313-ea97-4ee5-8714-65b3d368049e.png)

Reply:
That's good to know! I think your suspicion about the first assignment is right. In my case, the property was not reassigned a value and I didn't see the location of the assignment in the logs, just its value.

Reply:
> I wish MSBuild logs captured where a property was assigned its value, not just what its value is.

https://github.com/dotnet/msbuild/issues/2711. We need to figure out an efficient way to do it.